asv_bench/benchmarks/benchmarks_hindcast.py

```diff
@@ -0,0 +1,166 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+
+
+import dask
+import numpy as np
+import xarray as xr
+
+from climpred.bootstrap import bootstrap_hindcast
+# from climpred.constants import HINDCAST_COMPARISONS
```

Comment:
This can be updated with the circular imports PR merged.

Reply:
no. here I only want to test 'm2o' because `e2o` wouldnt take probabilistic